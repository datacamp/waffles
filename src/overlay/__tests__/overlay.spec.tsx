import { render } from '@testing-library/react';

import { Overlay } from '../index';

describe('Overlay', () => {
  it('renders snapshot when is visible', () => {
    const { container } = render(<Overlay isVisible={true} />);

    const overlay = container.querySelector('div');

    expect(overlay).toMatchSnapshot();
  });

  it('renders snapshot when is not visible', () => {
    const { container } = render(<Overlay isVisible={false} />);

    const overlay = container.querySelector('div');

    expect(overlay).toMatchSnapshot();
  });
});
