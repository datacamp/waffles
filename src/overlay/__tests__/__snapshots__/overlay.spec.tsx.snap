// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay renders snapshot when is not visible 1`] = `
@keyframes animation-0 {
  from {
    opacity: 1;
  }

  to {
    opacity: 0;
  }
}

.emotion-0 {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100vh;
  background-color: rgba(5, 25, 45, 0.6);
  z-index: 8000;
  touch-action: none;
  opacity: 0;
  -webkit-animation: animation-0 200ms ease-out forwards;
  animation: animation-0 200ms ease-out forwards;
}

<div
  class="emotion-0"
/>
`;

exports[`Overlay renders snapshot when is visible 1`] = `
@keyframes animation-0 {
  from {
    opacity: 0;
  }

  to {
    opacity: 1;
  }
}

.emotion-0 {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100vh;
  background-color: rgba(5, 25, 45, 0.6);
  z-index: 8000;
  touch-action: none;
  opacity: 0;
  -webkit-animation: animation-0 200ms ease-out forwards;
  animation: animation-0 200ms ease-out forwards;
}

<div
  class="emotion-0"
/>
`;
